### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2023-11-16

### DISCUSSION THREAD

**Discussion by EzraBrooks** [space-ros/space-ros#113] [General]
Previous meeting: #96 

### Agenda
- [ ] Prioritize issues that are currently marked with the next release's milestone

**Comment by ivanperez-keera** [comment]
I'd like to add to the agenda for next time reviewing any items left over from the October release (e.g., tagging the release in docker hub).

**Comment by EzraBrooks** [comment]
Action item:

- [ ] post new discussions and triaged issues to Discourse
- [ ] have a meeting about using GitHub issues/projects for prioritization

**Reply by EzraBrooks** [reply]
track down canadarm2 bug fix and make sure it gets released

**Comment by EzraBrooks** [comment]
Next meeting: #115
